ethereum/test/helpers.js

```diff
@@ -133,16 +138,18 @@ function signatureSubstrateToEthereum(sig) {
   return res;
 }
 
-function createMerkleTree(leavesHex) {
-  const leavesHashed = leavesHex.map(leaf => keccakFromHexString(leaf));
-  const merkleTree = new MerkleTree(leavesHashed, keccak, { sort: true });
-
+function createMerkleTree(values) {
+  const leaves = values.map(value => keccakFromHexString(value));
+  const merkleTree = new MerkleTree(leaves, keccak, {
+    sortLeaves: true,
```

Comment:
Should not be necessary. I think it makes no difference here since we do also sort the wallets, see my comment above. Let's remove this here to not create assumptions that won't hold in the actual production setup.

Reply:
will do when integrating real production data - added to trello